builder/dockerfile/internals.go

```diff
@@ -107,10 +111,17 @@ func (b *Builder) exportImage(state *dispatchState, imageMount *imageMount, runC
 func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error {
 	srcHash := getSourceHashFromInfos(inst.infos)
 
+	var commentStr string
+	if inst.flChown != "" {
+		commentStr = fmt.Sprintf("%s --chown=%s %s in %s ", inst.cmdName, inst.flChown, srcHash, inst.dest)
+	} else {
+		commentStr = fmt.Sprintf("%s %s in %s ", inst.cmdName, srcHash, inst.dest)
```

Comment:
minor nit, these lines are mostly the same so this could just be a variable for `chownComment` and use `%s %s%s %s in %s` as the format.

Could even be a separate function to keep it from cluttering the logic in `performCopy()`